website/src/parsers/css/stylis.js

```diff
@@ -0,0 +1,61 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'stylis/package.json';
+
+const ID = 'stylis';
+
+function removeLength(node, lineLengths) {
```

Comment:
What's the purpose of this?

Reply:
It's a workaround - if `length:0` is present astexplorer will hide all children